src/utils/metrics/reporter.go

```diff
@@ -44,37 +44,33 @@ func NewConsoleReporter(target io.Writer) Reporter {
 }
 
 func (r *ConsoleReporter) WriteSummary(metrics *Metrics) {
-	w := tabwriter.NewWriter(r.target, 1, 1, 1, ' ', tabwriter.AlignRight)
-
-	defer w.Flush()
-
+	writer := tabwriter.NewWriter(r.target, 1, 1, 1, ' ', tabwriter.AlignRight)
 	stats, totals := metrics.SumAllStats()
 
-	fmt.Fprintln(w, "\n --- Traffic stats ---")
-	fmt.Fprintf(w, "|\tTarget\t|\tRequests attempted\t|\tRequests sent\t|\tResponses received\t|\tData sent \t|\n")
+	defer writer.Flush()
```

Comment:
why did you move flush after SumAllStats()?

Reply:
just to combine variable declarations together